Lib/collections/__init__.py

```diff
@@ -876,6 +888,92 @@ def __iand__(self, other):
                 self[elem] = other_count
         return self._keep_positive()
 
+    def isequal(self, other):
+        ''' Test whether counts agree exactly.
+
+        Negative or missing counts are treated as zero.
+
+        This is different than the inherited __eq__() method which
+        treats negative or missing counts as distinct from zero:
+
+            >>> Counter(a=1, b=0).isequal(Counter(a=1))
+            True
+            >>> Counter(a=1, b=0) == Counter(a=1)
+            False
+
+        Logically equivalent to:  +self == +other
+        '''
+        if not isinstance(other, Counter):
+            other = Counter(other)
+        for elem in set(self) | set(other):
```

Comment:
Would this be a bit more readable:

```python
for elem in set(self) | set(other):
    left = max(self[elem], 0)
    right = max(other[elem], 0)
    if left != right:
        return False
return True
```

?

Reply:
Also slower ;-)